Rubberduck.VBEEditor/Selection.cs

```diff
@@ -68,19 +62,15 @@ public bool Contains(Selection selection)
             return false;
         }
 
-        private readonly int _startLine;
-        public int StartLine { get { return _startLine; } }
-        
-        private readonly int _endLine;
-        public int EndLine { get { return _endLine; } }
+        public int StartLine { get; }
+
+        public int EndLine { get; }
+
+        public int StartColumn { get; }
 
-        private readonly int _startColumn;
-        public int StartColumn { get { return _startColumn; } }
-        
-        private readonly int _endColumn;
-        public int EndColumn { get { return _endColumn; } }
+        public int EndColumn { get; }
 
-        public int LineCount { get { return _endLine - _startLine + 1; } }
+        public int LineCount => EndLine - StartLine + 1;
```

Comment:
I really don't like expressions like those. I would much prefer that it's `(EndLine - StartLine) + 1`. Yes I know it may be considered redundant but in this case, it reduces the cognition load.  (and yes I'm the dolt who didn't follow his own advice)